File: spectra-serve-system/src/main/java/com/yangxj96/spectra/server/system/Application.java
package com.yangxj96.spectra.server.system;

import org.apache.dubbo.config.spring.context.annotation.EnableDubbo;
import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

/**
 * 主运行文件
 */
@EnableDubbo
@MapperScan("com.yangxj96.saas.server.system.mapper")
@EnableDiscoveryClient
@EnableCaching
@SpringBootApplication
public class Application {

    public static void main(String[] args) {
        SpringApplication.run(Application.class, args);
    }

}

File: yangxj96-serve/yangxj96-serve-gateway/src/main/kotlin/com/yangxj96/saas/server/gateway/Application.kt
package com.yangxj96.saas.server.gateway

import org.apache.dubbo.config.spring.context.annotation.EnableDubbo
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

/**
 * 前端网关的主启动类
 */
@EnableDubbo
@EnableDiscoveryClient
@SpringBootApplication(exclude = [DataSourceAutoConfiguration::class])
class Application


fun main(args: Array<String>) {
    runApplication<Application>(*args)
}